gcip/addons/container/jobs/kaniko.py

```diff
@@ -54,34 +53,16 @@ def execute(
         dockerfile (str, optional): Name of the dockerfile to use. File is relative to context. Defaults to "Dockerfile".
         enable_push (bool, optional): Enable push to container registry, disabled to allow subsequent jobs to act on container tarball.
             Defaults to False.
+        docker_client_config (Optional[DockerClientConfig], optional): Creates the Docker configuration file base on objects settings,
+            used by crane to authenticate against given registries. Defaults to None.
         verbosity (str, optional): Verbosity of kaniko logging. Defaults to "info".
-        ecr_login (bool): If ```ecr-login``` should be registered as ```credStore``` in the ```.docker/config.json```.
-            Mutually exclusive with `registry_user_env_var` and `registry_login_env_var`. Defaults to `False`.
-        registry_user_env_var (Optional[str]): If you have to login to the registry before the push, you have to provide
-            the name of the environment variable, which contains the username value, here.
-            **DO NOT PROVIDE THE USERNAME VALUE ITSELF!** This would be a security issue!
-            Mutually exclusive with `ecr_login`.
-            Defaults to `None` which skips the login attempt.
-        registry_login_env_var (Optional[str]): If you have to login to the registry before the push, you have to provide
-            the name of the environment variable, which contains the password or token, here.
-            **DO NOT PROVIDE THE LOGIN VALUE ITSELF!** This would be a security issue!
-            Mutually exclusive with `ecr_login`.
-            Defaults to `None` which skips the login attempt.
 
     Returns:
-        Job: gcip.Job will be returned to create container images with ```name=kaniko``` and ```namespace=execute```.
+        Job: gcip.Job will be returned to create container images with ```namespace=execute```.
```

Comment:
The `namespace` value needs to be aligned with the job configuration in (currently) line 129.

Reply:
Fixed that.